modules/nw-multus-add-pod.adoc

```diff
@@ -7,17 +7,36 @@ ifeval::["{context}" == "configuring-sr-iov"]
 :sriov:
 endif::[]
 
+ifeval::["{product-version}" == "4.3"]
+:bz:
+endif::[]
+ifeval::["{product-version}" == "4.4"]
+:bz:
+endif::[]
+ifeval::["{product-version}" == "4.5"]
+:bz:
+endif::[]
+
 [id="nw-multus-add-pod_{context}"]
 = Adding a Pod to an additional network
 
 You can add a Pod to an additional network. The Pod continues to send normal
 cluster related network traffic over the default network.
 
+If a Pod already exists, you cannot attach it to an additional network. Only when a Pod is created are any additional networks requested attached to the Pod.
+
 ifdef::sriov[]
 [NOTE]
 =====
-The Network Resources Injector will inject the `resource` parameter into the Pod CR automatically if a NetworkAttachmentDefinition CR associated with the SR-IOV CNI plug-in is specified.
+The Network Resources Injector will inject the `resource` field into the Pod custom resource (CR) automatically if a NetworkAttachmentDefinition CR associated with the SR-IOV Container Network Interface (CNI) plug-in is specified.
```

Comment:
s/will inject/injects/
Also, I question whether the word _automatically_ is necessary.
And is it still accurate to word it as follows?

> The Network Resources Injector injects the `resource` field into the Pod custom resource (CR) if the SR-IOV Container Network Interface (CNI) plug-in specifies a NetworkAttachmentDefinition CR.

Trying to clarify for the reader what is meant by "is specified".

Reply:
@bobfuru, unrelated, but as it happens Pod is part of k8s, so I am incorrect when referring to it as a CR. A Pod is just a resource or, like anything else, an object.

Reply:
@bobfuru, I think the goal is to explain this:

> If a NetworkAttachmentDefinition is managed by the SR-IOV Network Operator, the SR-IOV Network Resource Injector adds the `resource` field to the Pod object automatically.

At some point, this seemed important, probably because it was in the original contributed content. Maybe it's helpful when someone sees the field and doesn't know why it is there.